src/blocks/form/captcha/inspector.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars */
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+import { InspectorControls } from '@wordpress/block-editor';
+
+import {
+	PanelBody,
+	TextControl,
+	ToggleControl
+} from '@wordpress/components';
+
+const Inspector = ({
+	attributes,
+	setAttributes
+}) => {
+	return (
+		<InspectorControls>
+			<PanelBody
+				title={ __( 'Settings', 'otter-blocks' ) }
```

Comment:
It's empty.

Reply:
I wanted to add the Site Key as an attribute, and later I found a much better option.